### ISSUE TITLE
Skill Selector + Tasks

### ISSUE THREAD

**Issue by kevinra** [xbpeng/DeepMimic#65]
According to the paper, skill selector is trained without a task objective. Is it possible to train with the task objective? For example, I want the character to have flexibility to learn both *strike* task and *throw* task and an ability to execute arbitrary sequence of these on demand.

Thanks

**Comment by xbpeng** [comment]
Yup, it should be possible to add additional task objectives when training the skill selector policy.
